src/Formula/index.ts

```diff
@@ -150,11 +150,11 @@ const common: Data = {
           break
         case "critRate_":
           operands.push(percent(0.05, { key, prefix: "default" }),
-            lookup(hit.move, objectKeyMap(allMoves, move => customBonus[`${move}_critRate_`]), 0))
+            lookup(hit.move, objectKeyMap(allMoves, move => premod[`${move}_critRate_`]), 0))
```

Comment:
this is right before the actual accumulation of stats right before dmg calc, so instead of `premod`, this should be `total`